modules/debts_api/lib/debts_api/v0/fsr_form_transform/additional_data_calculator.rb

```diff
@@ -14,22 +14,23 @@ def initialize(form)
           @form = form
           @comments = @form.dig('additional_data', 'additional_comments')
           @expense_calculator = DebtsApi::V0::FsrFormTransform::ExpenseCalculator.build(@form)
+          @bankruptcy = @form.dig('additional_data', 'bankruptcy') || {}
         end
 
         def get_data
-          {
-            'bankruptcy' => get_bankruptcy_data,
-            'additionalComments' => get_comments
-          }
+          data = { 'bankruptcy' => get_bankruptcy_data }
+          data['additionalComments'] = get_comments unless get_comments == "\n"
+          data
         end
 
         def get_bankruptcy_data
-          # these should probably be digs
+          return {} if @bankruptcy.empty?
```

Comment:
Is it possible that @bankruptcy could return nil, empty strings, etc? In that case, I would recommend `.blank?` instead which covers - nil, false, empty strings, strings with only whitespace, empty arrays, and empty hashes.